### PR TITLE
Create Browsh.yml

### PR DESCRIPTION
**Pull request by tobimensch**
Add Browsh project

### CODE REVIEW

File: _data/projects/Browsh.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- Firefox
+- remote
+- bandwidth
+- SSH
+- MoSH
+- web
+- command line
+upforgrabs:
+  name: help needed
+  link: https://github.com/browsh-org/browsh/issues/
```
Comment (ritwik12) [inline _data/projects/Browsh.yml]: @tobimensch Please change this to https://github.com/browsh-org/browsh/labels/help%20wanted

**Review by ritwik12 — CHANGES_REQUESTED** [review]

**Comment by tobimensch** [comment]
@ritwik12 
Is this good?

**Comment by ritwik12** [comment]
@tobimensch Yes it is

**Review by ritwik12 — APPROVED** [review]

**Comment by ritwik12** [comment]
@tobimensch Thanks!

**Pull request merged**